index.html

```diff
@@ -0,0 +1,87 @@
+    <title>Mi hoja de vida</title>
+    <!-- Enlace al archivo CSS -->
+    <link rel="stylesheet" href="styles.css">
+    <!-- Enlace a la fuente de texto -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <!-- Encabezado con tu nombre -->
+    <header>
+        <h1>Felipe Abella Ballesteros</h1>
+        <img src="fotoperfil.jpg" alt="Profile pic" class="profile-img">
+    </header>
+
+    <!-- Sección de información personal -->
+    <section id="personal_info">
+        <h2>Información Personal</h2>
+        <p>Dirección: Calle 188 #55a - 61 Int. 4 Apto. 303, Bogotá</p>
+        <p>Teléfono: +57 302 5722700</p>
+        <p>Correo electrónico personal: abellafelipe07@gmail.com</p>
+        <p>Correo electrónico institucional: felipeabba@unisabana.edu.co</p>
+    </section>
+
+    <!-- Sección de educación -->
+    <section id="education">
+        <h2>Educación</h2>
+        <ol>
+            <li>Liceo de Cervantes: <strong> 2009 - 2010 </strong></li>
+            <li>Colegio San Carlos: <strong> 2010 - 2022 </strong><br> <strong> Bachiller </strong> </li>
+            <li>Universidad de La Sabana: <strong> 2022 - actualidad</strong> <br> <strong> Estudiante de Ingeniería Informática </strong> </li>
+        </ol>
+    </section>
+
+    <!-- Sección de experiencia laboral -->
+    <section id="laboral_experience">
+        <h2>Experiencia Laboral</h2>
+        <ul>
+            <li>
+                <p>Tutor de matemática (independiente) - Tutor <br></p>
+                <strong> 2023 - actualidad </strong>
+            </li>
+        </ul>
+    </section>
+
+    <!-- Sección de habilidades -->
+    <section id="habilities">
+        <h2>Habilidades</h2>
+        <ul>
+            <li>Trabajo en equipo</li>
+            <li>Determinado</li>
+            <li>Perseverancia</li>
+        </ul>
+    </section>
+
+    <!-- Sección de intereses o hobbies -->
+    <section id="hobbies">
+        <h2>Hobbies</h2>
+        <ul>
+            <li>Escuchar música</li>
+            <img src="https://storage.googleapis.com/pr-newsroom-wp/1/2023/12/Generic-FTR-headers_V10.jpg" class="img"/>
+            <li>Jugar y ver fútbol</li>
+            <img src="https://ichef.bbci.co.uk/ace/ws/640/cpsprodpb/968C/production/_119804583_gettyimages-955410340.jpg.webp" class="img"/>
+            <li>Jugar videojuegos</li>
+            <img src="https://images.ctfassets.net/rbl6nw8n2c6i/5BNkTpdOEFzF8aAwIZis97/381e3e39db7f34aa0d413ed8295dd051/gaming_article.png?fit=thumb" class="img"/>
+            <li>Salir con amigos</li>
+            <img src="https://arc-anglerfish-arc2-prod-elespectador.s3.amazonaws.com/public/MJN6AHXMZ5EIBFKIF265AM2O4E.jpg" class="img"/>
+        </ul>

```

Comment:
```suggestion
        <ul>
            <li>
               <p>Escuchar música</p>
               <img src="https://storage.googleapis.com/pr-newsroom-wp/1/2023/12/Generic-FTR-headers_V10.jpg" class="img"/>
            </li>
        </ul>

```

Si cada elemento de la lista tiene su foto, lo más adecuado es que la foto también esté dentro del <li>